### ISSUE TITLE
doesnt work for higher versions

### ISSUE THREAD

**Issue by xhudik** [jupyterhub/jupyterlab-hub#6]
Hi there, we tried to install:
```
sudo jupyter labextension enable jupyterhub_labextension --py --sys-prefix
Enabling extension jupyterhub_labextension...
Enabling: jupyterhub_labextension
- Writing config: /opt/local/Library/Frameworks/Python.framework/Versions/3.5/etc/jupyter/labconfig
        X is not version compatible with installed JupyterLab version 0.17.5
            Expects JupyterLab version 0.17.0 from packaged module jupyterhub_labextension@0.1.0/lib/plugin.js
```

Which says that we have JupyterLab 0.17.5 but we need to donwgraded to 0.17. Is it intentional? I'd guess higher version is ok...

**Comment by willingc** [comment]
Hi @xhudik, The current build [here](https://github.com/jupyterhub/jupyterhub-labextension/blob/master/package.json#L21) is pinned to 0.17.0.

Since JupyterLab is still in alpha, I'll defer to @danielballan whether 0.17.5 is currently compatible or if changes need to be made here. Thanks!

**Comment by danielballan** [comment]
Quick answer: I haven't checked on 0.17.5 specifically yet. If you update 0.17.0 -> 0.17.5 in package.json, it may just work. I'll report back when I get a chance to upgrade and test.

**Comment by ellisonbg** [comment]
I have fixed this for JupyterLab 0.21, will update instructions further. Closing...
